pkg/satellite/service.go

```diff
@@ -395,22 +395,18 @@ func (s *Service) DeleteProjectMembers(ctx context.Context, projectID uuid.UUID,
 }
 
 // GetProjectMembers returns ProjectMembers for given Project
-func (s *Service) GetProjectMembers(ctx context.Context, projectID uuid.UUID, limit int, offset int64) (pm []ProjectMember, err error) {
+func (s *Service) GetProjectMembers(ctx context.Context, projectID uuid.UUID, pagination Pagination) (pm []ProjectMember, err error) {
 	defer mon.Task()(&ctx)(&err)
 	_, err = GetAuth(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	if limit < 0 || offset < 0 {
-		return nil, errs.New("invalid pagination argument")
+	if pagination.Limit > maxLimit {
```

Comment:
What happens when limit is negative or not supplied?

Reply:
@egonelbre this check is moved to repository

Reply:
When the client requests without limit parameter doesn't it default to `0` making it not return any results?

Reply:
yes. Should we return an error?

Reply:
If `limit` is optional, then we should default to a reasonable value. If it's not optional then return an error.